argparse/argparse.py

```diff
@@ -140,11 +151,9 @@ def parse_args(self, args=None):
 
     def _parse_args(self, args):
         # add optional args with defaults
-        arg_dest = []
```

Comment:
I chose to keep _Arg() the same and store one instance of it for each valid option string passed to add_argument(). Without the changes below this module does not behave like CPython's argparse when parsing options having both short and long option.